exir/emit/_emitter.py

```diff
@@ -1575,7 +1575,8 @@ def _find_fqn_for_placeholder(
                     warnings.warn(
                         "Mutation on a buffer in the model is detected. ExecuTorch assumes "
                         "buffers that are mutated in the graph have a meaningless initial state, "
-                        "only the shape and dtype will be serialized.",
+                        "only the shape and dtype will be serialized, unless a pass which marks "
+                        "spec.const=True such as InitializedMutableBufferPass is run.",
```

Comment:
can you just mention meta["et_init_buffer"] instead? spec.const is more of an implementation detail now that we have the pass.

Reply:
Yup that's a great point